content/custom-roles-and-permissions.md

```diff
@@ -6,19 +6,21 @@ metaDescription: "sensenet Custom Roles and Permissions"
 
 # Custom Roles and Permissions
 
+sensenet comes with a built-in role and permission system. These pre-defined role and permission type sets are adequate in most of the cases, but there might be some situation when it is desired to define custom ones.
+
 # Creating a new role
 
--> group creation
 
-# Modifying an existing role
+sensenet offers special and built-in groups and users by default. Built-in groups in sensenet are regular groups, meaning their membership can be freely modified. They are used in the default Content Repository structure extensively, but no hard-coded logic is built on them. This means you can freely change or replace these groups with your own custom groups in your permission settings.
 
--> editing groups
--> allow or deny permissions for groups
+There can be some cases when you need to create a new role, which is actually a new group. You just need to create the group and set the proper permission rules of this group, then add the users to it. This way instead of setting permissions to individual users one by one, they will earn the permissions of the group automatically. For example, if you would like to create the role "Approval", you create the group "Approval". Then in the subtree you allow the permission "Approve" and put the required users into the group.
```

Comment:
```suggestion
There can be some cases when you need to create a new role, which is actually a new group. You just need to create the group and set the proper permission rules of this group, then add the users to it. This way, instead of setting permissions to individual users one by one, they will earn the permissions of the group automatically. For example, if you would like to create the role "Approval", you create the group "Approval". Then in the subtree you allow the permission "Approve" and put the required users into the group.
```